### PR TITLE
add JinaEncoder class to handle Jina Embeddings

### PR DESCRIPTION
**Pull request by JoanFM**
Adding JinaEncoder class to allow users to use Jina Embeddings using Jina Embedding API https://jina.ai/embeddings/

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [ ] Infrastructure change (CI configs, etc)
- [ ] Non-code change (docs, etc)
- [ ] None of the above: (explain here)

## Test Plan

Unit tests have been added

### CODE REVIEW

**Comment by JoanFM** [comment]
Adding unit tests to class

**Comment by JoanFM** [comment]
> Adding unit tests to class

Done

**Review by igiloh-pinecone — CHANGES_REQUESTED** [review]
@JoanFM thank you very much for your contribution!

Please see only one comment about missing a system (==integration) test. Other than that LGTM

File: tests/unit/test_jina_encoder.py
[the remark this answers is not in the document]
Comment (JoanFM) [reply tests/unit/test_jina_encoder.py:1]: Hey @igiloh-pinecone ,

System test added as request. Please check

**Review by igiloh-pinecone — APPROVED** [review]
LGTM!

**Pull request merged**